app/generator/content/part1.js

```diff
@@ -0,0 +1,7 @@
+const part1 = (_statement) => {
```

Comment:
do we need to pass statement in if we dnt use it?

Reply:
We will in the next ticket.  It's more just a placeholder to help whoever picks that up understand the flow.